config/kubernetes/sig-release/teams.yaml

```diff
@@ -163,6 +163,22 @@ teams:
     privacy: closed
     previously:
     - kubernetes-release-managers
+  release-manager-associates:
+    description: People are apprentices to the Branch Managers, formerly referred
```

Comment:
How about:

_People who are apprentices for branch management, formerly referred to as branch manager shadows._

or

_People shadowing branch management._

Reply:
that sounds better, thank you!